sdk/core/azure-core/src/main/java/com/azure/core/models/PollResult.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.models;
```

Comment:
Or package `com.azure.core.util.polling`?

Reply:
I think it should go to `polling`, what do you think @alzimmermsft  @srnagar

Reply:
Yeah, this should be in `com.azure.core.util.polling` along with the rest of the models for polling.

Reply:
Moved to `com.azure.core.util.polling`